src/odemis/driver/test/simcam_test.py

```diff
@@ -31,18 +31,28 @@
 
 CLASS = simcam.Camera
 KWARGS_FOCUS = {"name": "focus", "role": "overview-focus", "axes": ["z"], "ranges": {"z": [0, 0.012]}}
-KWARGS = dict(name="camera", role="overview", image="simcam-fake-overview.h5")
+
+KWARGS_STAGE = {"name": "stage", "role": "stage", "axes": ["x","y"], "ranges": {"x": [0, 0.013],"y": [0, 0.013]}}
+#KWARGS = dict(name="camera", role="overview", image="/home/zaabouti/Downloads/FM-grid/tile.tif", resolution=[128, 128])
+KWARGS = dict(name="camera", role="overview", image="9.tif", resolution=[512, 512])
```

Comment:
Drop the commented line.
Also, "9.tif" doesn't exists. I'd rather not introduce even more test images in the Odemis source code. Maybe you can just use "songbird-sim-ccd.h5", which has MD_WL_IN.